src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java

```diff
@@ -110,16 +110,15 @@ public ResponseEntity<Page<AdvertisementTitleDto>> getPageOfAdvertisementsByKeyw
     @Operation(summary = "Find the required number of advertisements for the selected category and return it as a page result")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "OK"),
-            @ApiResponse(responseCode = "400", description = "BAD REQUEST"),
-            @ApiResponse(responseCode = "404", description = "NOT FOUND")})
+            @ApiResponse(responseCode = "400", description = "BAD REQUEST")})
     public Page<AdvertisementTitleDto> findAdvertisementsHavingCategory(
             @PathVariable("category_id") Long categoryId,
```

Comment:
You can annotate that as Positive in addition to verifying the category for existence.

Reply:
added